lib/Wikia/src/Service/Gateway/GatewayBase.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Wikia\Service\Gateway;
+
+use Http;
+
+class GatewayBase {
```

Comment:
I wonder if we still need this class, since for now we'll be relying on Swagger generated code to take care of the http plumbing for us